llvm/test/Transforms/SROA/basictest.ll

```diff
@@ -1992,8 +1992,9 @@ define i32 @load_volatile_past_end() {
 
 define i32 @load_atomic_past_end() {
 ; CHECK-LABEL: @load_atomic_past_end(
-; CHECK-NEXT:    [[A_0_LOAD_EXT:%.*]] = zext i1 undef to i32
-; CHECK-NEXT:    ret i32 [[A_0_LOAD_EXT]]
+; CHECK-NEXT:    [[A:%.*]] = alloca i1, align 1
+; CHECK-NEXT:    [[V:%.*]] = load atomic i32, ptr [[A]] seq_cst, align 4
+; CHECK-NEXT:    ret i32 [[V]]
```

Comment:
Due to this test, we may want to check only the type without `DL`. Adding too many checks hinders optimization.